v2/pcb-v2.1/rgb-led-v2.net

```diff
@@ -1,92 +1,124 @@
 (export (version D)
   (design
-    (source /mnt/data/Projects/rgb-led/v2/pcb-v2/rgb-led-v2.sch)
-    (date "Sun 30 Nov 2014 01:23:26 PM CET")
-    (tool "eeschema (2013-may-18)-stable"))
+    (source /home/roughl/proggen/projects/coredump/rgb-led/v2/pcb-v2.1/rgb-led-v2.sch)
+    (date "Don 15 Sep 2016 17:41:07 CEST")
+    (tool "Eeschema 4.0.3-stable")
+    (sheet (number 1) (name /) (tstamps /)
+      (title_block
+        (title "RGB LED v2")
```

Comment:
Is this the netlist? Why does it have all this metadata? :laughing: Anyways, you *could* update this to v2.1...